src/components/sidebar/nodeView/StopAreaView.tsx

```diff
@@ -174,13 +180,8 @@ class StopAreaView extends ViewFormBase<IStopAreaViewProps, IStopAreaViewState>
         } catch (e) {
             this.props.errorStore!.addError(`Tallennus epäonnistui`, e);
         }
-
-        if (this.props.isNewStopArea) {
-            this.navigateToNewStopArea();
-            return;
-        }
-        this.setState({ isLoading: false });
         this.props.stopAreaStore!.setIsEditingDisabled(true);
+        navigator.goBack();
```

Comment:
This should only be done if the save was successful.

Reply:
However, im not even sure if i like it going back. What is the user wants to keep editing after save? Maybe let user click <- instead? Not sure...

Reply:
There is not much to edit so I doubt the user wouldnt do all the changes at once. Though in other views saving something never redirects so I agree that stopAreaView shouldn't be exception to this.